t/plugin/request-id.t

```diff
@@ -437,71 +437,3 @@ GET /opentracing
 X-Request-ID: 123
 --- response_headers
 X-Request-ID: 123
-
-
-
-=== TEST 15: add plugin with algorithm nanoid (default uuid)
---- config
-    location /t {
-        content_by_lua_block {
-            local t = require("lib.test_admin").test
-            local http = require "resty.http"
-            local v = {}
-            local ids = {}
-            local code, body = t('/apisix/admin/routes/1',
-                 ngx.HTTP_PUT,
-                 [[{
-                        "plugins": {
-                            "request-id": {
-                                "algorithm": "nanoid"
```

Comment:
your PR suggests that usage method/configuration for nanoid has changed. Which means existing users will face incompatibility issues when upgrading to the newer version. We should avoid such a situation.

Reply:
I have noticed the question you raised before submitting. I have already set the initial default value in the parameters, the default parameter value will be the same as the default value generated in previous versions without modifying the parameter value. Therefore, I think the upgrade should be smooth. Is there anything missing from me?